example/tools/lint/linters.bzl

```diff
@@ -118,3 +119,9 @@ spotbugs = lint_spotbugs_aspect(
 )
 
 spotbugs_test = lint_test(aspect = spotbugs)
+
+keep_sorted = lint_keep_sorted_aspect(
+    binary = "@@gazelle~~go_deps~com_github_google_keep_sorted//:keep-sorted",
```

Comment:
I'll fix this up - you shouldn't rely on the canonical name (for example the `~` changed to a `+`

should always use the apparent name and let Bazel resolve it.